### ISSUE TITLE
iojs: does not compile with io.js v2.0.0

### ISSUE THREAD

**Issue by imyller** [wadey/node-microtime#27]
With nan `1.7.0` and io.js `2.0.0` microtime does not compile:

```
In file included from ../src/microtime.cc:11:
In file included from ../node_modules/nan/nan.h:80:
In file included from ../node_modules/nan/nan_new.h:190:
../node_modules/nan/nan_implementation_12_inl.h:181:66: error: too many arguments to function call, expected at most 2, have 4
  return v8::Signature::New(v8::Isolate::GetCurrent(), receiver, argc, argv);
         ~~~~~~~~~~~~~~~~~~                                      ^~~~~~~~~~
/Users/-/.node-gyp/2.0.0/deps/v8/include/v8.h:4188:3: note: 'New' declared here
  static Local<Signature> New(
  ^
```
 
With nan `1.8.4` and io.js `2.0.0` microtime compiles, but gives deprecation warnings:
 

```
  CXX(target) Release/obj.target/microtime/src/microtime.o
../src/microtime.cc:51:36: warning: 'ErrnoException' is deprecated: Use UVException(isolate, ...) [-Wdeprecated-declarations]
        return NanThrowError(node::ErrnoException(errno, "gettimeofday"));
                                   ^
/Users/-/.node-gyp/2.0.0/src/node.h:81:45: note: 'ErrnoException' has been explicitly marked deprecated here
                inline v8::Local<v8::Value> ErrnoException(
                                            ^
/Users/-/.node-gyp/2.0.0/src/node.h:47:42: note: expanded from macro 'NODE_DEPRECATED'
    __attribute__((deprecated(message))) declarator
  
...

3 warnings generated.
  SOLINK_MODULE(target) Release/microtime.node
  SOLINK_MODULE(target) Release/microtime.node: Finished
```

**Comment by imyller** [comment]
Fixed in `1.4.0`

**Comment by wadey** [comment]
1.4.1 fixes the deprecation warnings
